Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -358,10 +358,16 @@ define([
         content._rtcCenterTransform = Matrix4.clone(Matrix4.IDENTITY);
         var rtcCenter = featureTable.getGlobalProperty('RTC_CENTER', ComponentDatatype.FLOAT, 3);
         if (defined(rtcCenter)) {
-            content._rtcCenterTransform = Matrix4.fromTranslation(Cartesian3.fromArray(rtcCenter), content._rtcCenterTransform);
+            content._rtcCenterTransform = Transforms.eastNorthUpToFixedFrame(Cartesian3.fromArray(rtcCenter), content._rtcCenterTransform);
+            tile._hasRTC = true;
         }
 
+        // Use the boundingSphere fallback if the root tile has neither a defined transform nor an RTC property. 
+        // If the root tile hasn't loaded yet, we don't know if it has RTC, so we assume it does not.
+        tile._useBoundingSphereForClipping = !defined(tileset.root._header.transform) && !defined(tileset.root._hasRTC);
```

Comment:
Should we check each individual tile for RTC? It's defined per-tile in the spec, not on the tileset itself.

Reply:
This is a design decision. I think the most straightforward design is to say that the clipping plane is relative to the root tile.

This is to resolve the situation when child tiles have different RTC's, which one do you use? For transforms, we only use the root tile's, so this seemed like the consistent choice.

Reply:
Sorry, was mistaken, this approach just checking the root is fine.